feerates/2020-04--2022-04-feerates.csv

```diff
@@ -1,4 +1,4 @@
-﻿0.00016499
```

Comment:
interesting that this is literally invisible on GitHub

Reply:
Aye.